SYCL/lit.cfg.py

```diff
@@ -368,6 +368,8 @@
         if lit_config.params.get('ze_debug'):
             gpu_run_substitute = " env ZE_DEBUG={ZE_DEBUG} SYCL_DEVICE_FILTER=level_zero:gpu,host ".format(ZE_DEBUG=config.ze_debug)
             config.available_features.add('ze_debug'+config.ze_debug)
+    elif config.sycl_be == "ext_intel_esimd_emulator":
+        gpu_run_substitute += "CM_RT_PLATFORM=skl "
```

Comment:
please add comments what this setting means - `CM_RT_PLATFORM=skl`

Reply:
Done.